### ISSUE TITLE
Poor finetuning accuracy

### ISSUE THREAD

**Issue by Zumbalamambo** [davidsandberg/facenet#895]
I excluded the Logits layer and finetuned the model with 40 identities. The resulting model has got poor accuracy. It detects even unknown person as 100% known person. why so?
